src/common/registry-csm/fetch/operator.fetch.ts

```diff
@@ -40,6 +40,18 @@ export class RegistryOperatorFetchService {
       return true;
     }
 
+    const nodeOperatorRewardAddressChangedFilter =
```

Comment:
I think we need to add this event handling for  curated kind of modules too

Reply:
This is already provided, just different event names

NodeOperatorRewardAddressChanged — in CSM
NodeOperatorRewardAddressSet — in NOR